tests/pytorch_tests/model_tests/feature_models/manual_bit_selection.py

```diff
@@ -218,8 +219,7 @@ def get_tpc(self):
         return {'mixed_precision_activation_model': tpc}
 
     def get_resource_utilization(self):
-        return mct.core.ResourceUtilization(activation_memory=6200)
-
+        return mct.core.ResourceUtilization(activation_memory=15000)
 
```

Comment:
why this change? is it just to prevent auto mp execution?

Reply:
It's larger because max cut is larger than max tensor.